pkg/backend/httpstate/backend.go

```diff
@@ -202,7 +202,7 @@ var _ backend.SpecificDeploymentExporter = &cloudBackend{}
 
 // New creates a new Pulumi backend for the given cloud API URL and token.
 func New(d diag.Sink, cloudURL string, project *workspace.Project, insecure bool) (Backend, error) {
-	cloudURL = ValueOrDefaultURL(cloudURL)
+	cloudURL = ValueOrDefaultURL(pkgWorkspace.Instance, cloudURL)
```

Comment:
Trying to lift these higher, cascades out to quite a few things though.

Reply:
Will you follow up with that, or do we need a ticket & TODO comment for this?

Reply:
I'll follow up. I want to add a linter to error out on any use of "pkgWorkspace.Instance" anywhere but main.go, so these will get cleaned up.